### ISSUE TITLE
Some timeline animations should be possible on dynamic physics bodies

### ISSUE THREAD

**Issue by Ben-G** [spritebuilder/SpriteBuilder#616]
In version 1.1 SB forces a node with enabled physics to be static as soon as any timeline animation is applied. However, some of the timeline animations such as sprite frames, color and opacity don't influence the position/orientation of the node and should be applicable for nodes with dynamic physics bodies, too.

It should be possible to animate a character with sprite frames even if this character has a dynamic physics body.

**Comment by Ben-G** [comment]
With `CCAnimation` being retired this is a fairly major issue. Does this mean there's currently no accepted way of creating a sprite frame animation for a physics object? Or am I missing something?

**Comment by cocojoe** [comment]
Agreed that it seems only fair to be able to have sprite frame animation with a dynamic physics body.  I'll have a look next week.
